packages/theming/atlas/tests/e2e/allScreens.spec.ts

```diff
@@ -0,0 +1,108 @@
+const fs = require("fs");
+const { join, resolve } = require("path");
+const cwd = process.cwd();
+const xmlParser = require("fast-xml-parser");
+
+// For every module mx exports a folder, and in that folder we have bunch of xml's
+// which might contain an URL field
+const testPageFolders = join(cwd, "tests/testProject/deployment/web/pages/en_US");
+const screenShotsFolder = join(cwd, "tests/e2e/screenshot-baseline");
+
+// TODO [https://mendix.atlassian.net/browse/WT-3106]: Cannot save big screens due to wdio-image-service/webdriver-image-comparison/canvas failiure
+// Need to keep the list until this fixed: https://github.com/wswebcreation/webdriver-image-comparison/issues/60
+const pagesToSkip = ["/p/chat-fullheight/{Id}", "/p/chat-variants/{Id}"];
+
+// Mostly the pages with progressbar fails since it is not CSS animations for web -_-. So disableCSSAnimation wont work.
+// This ends up having unstable progress circle percentage
+const pagesWithTimeout = [
+    "/p/datagrid-manyrows",
+    "/p/progresscircle",
+    "/p/pt_dashboard-expenses",
+    "/p/pt_dashboard-metrics",
+    "/p/pt_tablet_dashboard-metrics",
+    "/p/pt_tablet_dashboard-user-detail",
+    "/p/bb_headers",
+    "/p/bb_cards",
+    "/p/pt_dashboard-user-detail"
+];
+
+describe("Screenshots of the pages for", () => {
+    beforeAll(() => {
+        browser.url("/");
+    });
+
+    for (const url of pageUrls(testPageFolders)) {
+        if (url && !pagesToSkip.includes(url)) {
```

Comment:
Why needs the url be checked on falsyness here? For `""`?

Reply:
Removed